src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,9 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery[] balls = { new Lottery(), new Lottery(), new Lottery() };
```

Comment:
Instead of manually creating each `Lottery` instance, consider using a loop to create the instances. This approach is recommended in the checklist to avoid redundancy and improve code maintainability .